### PR TITLE
1415 snapshotcollector added

### PR DESCRIPTION
**Pull request by anton-abyzov**
### Problem
There is no good telemetry, exceptions logs

### Solution
SnapshotCollector nuget added

### Proposed of changes
Using nuget package connected to Azure

### CODE REVIEW

File: VirtoCommerce.Platform.Web/App_Start/FilterConfig.cs [hunk 1]
```diff
@@ -10,10 +10,10 @@ public class FilterConfig
     {
         public static void RegisterGlobalFilters(GlobalFilterCollection filters)
         {
-            filters.Add(new HandleErrorAttribute());
-			filters.Add(new AiHandleErrorAttribute());
+            //filters.Add(new ErrorHandler.AiHandleErrorAttribute());
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/App_Start/FilterConfig.cs]: Could you explain why you commented on this line?

File: VirtoCommerce.Platform.Web/ApplicationInsights.config [hunk 2]
```diff
@@ -70,38 +75,40 @@
...
+    <Add Type="Microsoft.ApplicationInsights.Web.AspNetDiagnosticTelemetryModule, Microsoft.AI.Web" />
+  </TelemetryModules>
+  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel" />
+  <!-- 
     Learn more about Application Insights configuration with ApplicationInsights.config here: 
     http://go.microsoft.com/fwlink/?LinkID=513840
     
     Note: If not present, please add <InstrumentationKey>Your Key</InstrumentationKey> to the top of this file.
-  --></ApplicationInsights>
+  -->
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/ApplicationInsights.config]: Please remove your InstrumentationKey from public config

File: VirtoCommerce.Platform.Web/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -28,6 +28,18 @@ j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
     <base href="~/" />
     @Html.Partial("_FacebookTrackerPartial")
     @Html.Partial("_AppInsightsPartial")
+    <script type = 'text/javascript' >
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Views/Shared/_Layout.cshtml]: This UI snippet is redundant and not related to this task

File: VirtoCommerce.Platform.Web/VirtoCommerce.Platform.Web.csproj [hunk 4]
```diff
@@ -29,6 +29,8 @@
     <TargetFrameworkProfile />
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <ApplicationInsightsResourceId>/subscriptions/dc56b6b7-adb2-402d-8637-2c248db98686/resourcegroups/Default-ApplicationInsights-EastUS/providers/microsoft.insights/components/VirtoCommerce.Platform.Web</ApplicationInsightsResourceId>
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/VirtoCommerce.Platform.Web.csproj]: These lines are client specific and can be removed from project file

File: VirtoCommerce.Platform.Web/Web.config [hunk 5]
```diff
@@ -11,11 +11,11 @@
         <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog" />
     </configSections>
     <common>
-        <logging>
+        <!--<logging>
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Web.config]: Why you've commented out this section?

File: VirtoCommerce.Platform.Web/App_Start/FilterConfig.cs (hunk 1 above)
Comment (anton-abyzov) [reply VirtoCommerce.Platform.Web/App_Start/FilterConfig.cs]: The line is exactly the same as above 			filters.Add(new AiHandleErrorAttribute());

**Comment by anton-abyzov** [comment]
Removed all personal info left from connection of my own Azure account (linking with AppInsights). All the rest is needed for nuget package to work, and settings are there now as well

**Pull request merged**